docs/source/developers/contributing/known-issues.rst

```diff
@@ -1,21 +0,0 @@
-.. _knownissues:
-
-Known Issues
--------------
-
-``gopls`` Linting Issue in Testing Files on Package Declaration Line
```

Comment:
Should we still have a record of this somewhere?

Reply:
Yea I actually put this under the VS Code setup. I realized t's not so much of an issue but rather a config.

<img width="2032" alt="image" src="https://github.com/user-attachments/assets/f8166178-1369-4da1-bd09-0a89af9cafb4">